services/cal/event.py

```diff
@@ -2,24 +2,23 @@
 from datetime import timedelta, datetime as dt
 from icalendar import Event as iCalEvent
 from icalendar import Alarm, vDatetime
+import pytz
 
 class Event(iCalEvent):
+    local_tz = pytz.timezone('Europe/Berlin')
```

Comment:
Timezone hardcoded ?

Reply:
Managed to do the same without. `.astimezone()` can be used to get system timezone.